clientV2/assets/components/_modal.scss

```diff
@@ -0,0 +1,68 @@
+$primaryGray: #777777;
+
+.modal {
+  top: 0;
+  left: 0;
+  height: 100vh;
+  right: 0;
+  background-color: rgba(54, 93, 220, 0.3);
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  z-index: 1040;
+
+  .modal_content {
+    background-color: #FFF;
+    box-shadow: 0px 1px 4px rgba(0, 0, 0, 0.25);
+    min-width: 480px;
+    max-width: 80%;
+    width: auto;
+    overflow-y: auto;
+    border-radius: 10px;
+
+    @media (min-width: 600px) {
+      padding: rem(60px);
+      max-width: rem(634px);
+    }
+  }
+
+  .modal_header {
+    border-bottom: 1px solid rgba(0, 0, 0, 0.06);
+    display: flex;
+    justify-content: space-between;
+    padding: 10px 20px 0px 20px;
+
+    p {
+      font-family: 'Open Sans', sans-serif;
+      font-style: normal;
+      font-size: 28px;
+      font-weight: 100;
+      margin: 0px;
+      color: $primaryGray;
+    }
+
+    button {
+      border: none;
+      font-size: 35px;
+      font-weight: 100;
+      color: $primaryGray;
+      font-family: 'Open Sans', sans-serif;
+      &:focus {
+        outline: none;
+      }
+    }
+  }
+
+  .modal_body {
+    padding: 25px;
+    button {
```

Comment:
Why are you writing a style for `button` that is not in you `.modal_body`` div`. I think there is no need for this button style since there is no button in your JSX markup. Also a reuseable Button component will be created which will come with its own style